app/src/main/res/layouts/Activity/Product/layout/activity_home.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file (activity_home.xml) should be in app\src\main\res\layouts\Activity\layout